openhtf/__init__.py

```diff
@@ -201,7 +226,7 @@ def PhaseTwo(test):
                     FLAGS.http_port,
                     os.getpid()).SaveToFile(FLAGS.rundir)
   
-    logging.info('Executing test: %s', self.filename)
+    _LOG.info('Executing test: %s', self.filename)
```

Comment:
Now that I'm looking at this, it should probably be something more like "Starting test execution" since this log line applies to multiple test runs.